backend/sirius-web-tests/src/test/java/org/eclipse/sirius/web/tests/GeneralPurposeTestCases.java

```diff
@@ -124,8 +124,8 @@ private File getRootFolder() {
             List<Path> filePaths = paths.filter(Files::isRegularFile)
                     .filter(filePath -> filePath.toFile().getName().endsWith(extension))
                     .filter(filePath -> !filePath.toString().replace("\\", "/").contains("/.mvn/wrapper/")) //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
-                    .filter(filePath -> !filePath.toString().contains("/sirius-web-domain")) //$NON-NLS-1$
-                    .filter(filePath -> !filePath.toString().contains("/sirius-web-view")) //$NON-NLS-1$
+                    .filter(filePath -> !filePath.toString().contains("sirius-web-domain")) //$NON-NLS-1$
+                    .filter(filePath -> !filePath.toString().contains("sirius-web-view")) //$NON-NLS-1$
```

Comment:
There's an example right on the previous line of how to handle these in a portable way.